R/bayesianmetaanalysis.R

```diff
@@ -1600,32 +1644,51 @@ BayesianMetaAnalysis <- function(jaspResults, dataset, options) {
 
 .bmaFillSeqPlot <- function(seqPlot, jaspResults, dataset, options, .bmaDependencies, type){
   
+  evidenceR <- gettext("Evidence for random effects")
+  evidenceF <- gettext("Evidence for fixed effects")  
+    
   rowResults <- .bmaSequentialResults(jaspResults, dataset, options, .bmaDependencies)
   if(type == "ES"){
     BFs <- rowResults$BFs
   } else if(type == "SE"){
+  # The BFs for heterogeneity have different labels
     BFs <- rowResults$BFsHeterogeneity
     yName <- "BF[italic(rf)]"
+    pizzaTxt <- c("data | f", "data | r")
+    bfSubscripts <-  c("BF[italic(rf)]", "BF[italic(fr)]")  
+    arrowLabel <- c(evidenceF, evidenceR)
+    BF <- BFs[length(BFs)]
+    if(BF >= 1){
+      modelEvidence <- gettext("random")
+    } else {
+      modelEvidence <- gettext("fixed")
+    }
+    allEvidenceLabels <- c(gettext("Anecdotal",domain="R-jaspGraphs"), 
+                           gettext("Moderate",domain="R-jaspGraphs"),
+                           gettext("Strong",domain="R-jaspGraphs"), 
+                           gettext("Very Strong",domain="R-jaspGraphs"), 
+                           gettext("Extreme",domain="R-jaspGraphs"))
+    if(BF < 1) BF     <- 1/BF
+    idx               <- findInterval(BF, c(1, 3, 10, 30, 100), rightmost.closed = FALSE)
```

Comment:
So this generates the following plot:
![Screenshot 2021-04-22 at 18 04 06](https://user-images.githubusercontent.com/7093546/115746985-228c3280-a395-11eb-8bf1-2302c3e42658.png)

I don't know whether this is correct, because I don't know the convention for labelling the hypotheses, but the `data | r H1` and `data | f H1` seems off (should it not be `data | Hr`, `data | Hf`)? If it's correct as it is, ignore this comment.

Reply:
I agree with Simon. I'm also a bit confused. Does it refer to the fixed effects and random effects model? There are various H1 in the game. Perhaps it's more direct to refer to them as FE and RE.

Reply:
Hmm I believe that FE H1 and RE H1 didn't fit (the last bit got cut from the pizza plot) so we chose this format. I could change it to FE and RE if you think it is clear that it regards the alternative hypotheses?

Reply:
Alternatively, you can also just number the models. Would that help? If there are 8 models involved, perhaps refer to them as H0, H1, H2, H3, H4, H5, H6, H7.